DECISIONLOG.md

```diff
@@ -1,9 +1,9 @@
-# dbt_Recharge_source v0.1.0
+# dbt_Recharge_source v0.1.0 
```

Comment:
This is not needed in the DECISIONLOG. Instead we can just have this be Decision Log. Similar to what we did in [Zendesk](https://github.com/fivetran/dbt_zendesk/blob/main/DECISIONLOG.md).